core/ir/aarch64/codec.c

```diff
@@ -298,6 +298,7 @@ decode_sysreg(uint imm15)
     case 0x5a10: sysreg = DR_REG_NZCV; break;
```

Comment:
We may want to mention this in the changelog at https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L138 (use `#` to make a doxygen link: e.g., `+ Added the AArch64 FPMR register as #DR_REG_FPMR`.

Reply:
I've added it, but without the link because if I try the link I get `explicit link request to 'DR_REG_FPMR' could not be resolved`. That seems to be because of #3728. I could make other changes in `opnd_api.h` to work around that doxygen problem but I think it's probably better to just do without the link in the changelog.